### DISCUSSION TITLE
TestFlight full

### DISCUSSION THREAD

**Discussion by djsnipa1** [mlemgroup/mormaer-mlem#195] [General]
I'd love to start using this app. The TestFlight beta is full. Any chance to get this on the AppStore soon? Thanks for the hard work!

**Comment by buresdv** [comment]
We have to implement all the basic features, otherwise Apple will reject the app.

Our planned release date is July 1st, so hopefully we'll make it!

**Comment by jessearmstrong** [comment]
Very much looking forward to this. I am enjoying Lemmy, but isn't very good on mobile.

**Comment by BrooklynMan** [comment]
a new TestFlight has opened up, and new slots are available!

Sign up here! https://testflight.apple.com/join/MelFP11Y
